### PR TITLE
Update banner design

### PR DESCRIPTION
**Pull request by Nikomus**
- Update donation banner design

https://neontribe.atlassian.net/browse/SEAP-134

### CODE REVIEW

File: src/assessment.handlebars [hunk 1]
```diff
@@ -210,7 +210,7 @@
   </div>
   <div class="slide are-you-sure" id="are-you-sure">
     <div class="box">
-      <h1 tabindex="-1" style="red">Are you sure?</h1>
```
Comment (katjam) [inline src/assessment.handlebars]: lol

Comment (Nikomus) [reply src/assessment.handlebars]: @katjam couldn't deal with it after you pointed it out 😅

File: src/assessment.handlebars [hunk 2]
```diff
@@ -1043,8 +1043,9 @@
   </div>
   <div class="slide main-menu" id="main-menu">
     <div class="flow">
-      <div class="center donation-banner">Text SEAP02 £2 to 70070 to help us support more people like you.
-          <img src="images/jtg_logo.png" alt="Just Text Giving logo"/>
+      <div class="donation-banner">
+        <p class="center">If you find this website useful, please consider donating.</p>
+        <p class="center">Text <b>'SEAP02 £2'</b> to <b>70070</b> to help us support more people like you.</p>
```
Comment (katjam) [inline src/assessment.handlebars]: Same as on esa - judgement of whether to use the html entity or leave it as it is. Not sure if the encoding is necessary anymore  - but may have been an accessibility enhancement - but would have to look into it.

Interesting: https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/

Comment (katjam) [reply src/assessment.handlebars]: https://www.levelaccess.com/csscontentproperty/

screenshot_20181109_120553

**Comment by Nikomus** [comment]
As this is the approved design and there is some inconsistency already with quote/apostrophe usage we are merging as is with a view to review again in a later release.

**Pull request merged**